app/main.py

```diff
@@ -38,3 +39,20 @@ async def index():
 def blocking():
     time.sleep(2)
     return {"ret": 42}
+
+
+# Get https://phinvads.cdc.gov/baseStu3/ValueSet and return response
+@app.get("/phinvads/ValueSet")
+@cache(namespace="phinvads", expire=3600)
+def value_set(
+    name: str = None, title: str = None, identifier: str = None, _getpages: str = None
+):
+    url = "https://phinvads.cdc.gov/baseStu3/ValueSet"
+    params = {
+        "name": name,
+        "title": title,
+        "identifier": identifier,
+        "_getpages": _getpages,
+    }
+    response = requests.get(url, params=params)
+    return response.json()
```

Comment:
Should we add in error handling now, or do we want to break that out into a separate ticket? (Seems like most errors are returned from PV as plain text, so when we hit this it throws a 500)

Reply:
ooo this is a good q. i did not add error handling on my open PR either, but open to doing that!

Reply:
Good callout! I will add it in :)